Rakefile

```diff
@@ -1,9 +1,17 @@
-require 'cucumber/rake/task'
-require 'rspec/core/rake_task'
-require 'rubocop/rake_task'
+require 'data_mapper'
+require './app/app.rb'
 
-RuboCop::RakeTask.new :cop
-RSpec::Core::RakeTask.new :spec
-Cucumber::Rake::Task.new :cuke
+namespace :db do
+  desc "Non destructive upgrade"
+  task :auto_upgrade do
+    DataMapper.auto_upgrade!
+    puts "Auto-upgrade complete (no data loss)"
+  end
 
-task default: [:cop, :spec, :cuke]
\ No newline at end of file
+
+  desc "Destructive upgrade"
+  task :auto_migrate do
+    DataMapper.auto_migrate!
+    puts "Auto-migrate complete (data was lost)"
```

Comment:
That could be a very scary message! I would maybe soften the wording a little bit (data may have been lost)